qiita_db/reference.py

```diff
@@ -171,6 +171,10 @@ def tree_fp(self):
                         JOIN qiita.{0} r ON r.tree_filepath=f.filepath_id
                      WHERE r.reference_id=%s""".format(self._table)
             qdb.sql_connection.TRN.add(sql, [self._id])
-            rel_path = qdb.sql_connection.TRN.execute_fetchlast()
+            try:
```

Comment:
The test exist cause the Silva reference and a biom table that uses it exists. Basically, when we try to call it we are testing that functionality without being explicit. I can see your confusion and that this is what we want to do in the future but not what we are doing now. Adding a test so we can merge and then discuss offline.

Now about the return. I really don't like having 2 different returns in the same function (None, string) as this will require some changes to make it work with other sections of the code. For example, when we create the summary of the artifact we display the value of the tree. Having an empty string make way more sense,